sponsorship/templates/sponsorship/create_profile.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{% block content %}
+    <h2>
+        Create Sponsorship Profile
+    </h2>
+    <form method="post" enctype="multipart/form-data">
+        {% csrf_token %}
+        {{ form.as_p }}
+        <button type="submit">
+            Submit
+        </button>
+    </form>
+{% endblock body %}
```

Comment:
```suggestion
{% endblock content %}
```

To match with `{%block content %}` in line 2